readme.md

```diff
@@ -370,6 +378,9 @@ If you are new to eBPF, you may want to try the links described as "introduction
 - [ebpfkit-monitor](https://github.com/Gui774ume/ebpfkit-monitor) - An utility to statically analyze eBPF bytecode or monitor suspicious eBPF activity at runtime. It was specifically designed to detect ebpfkit.
 - [Bad BPF](https://github.com/pathtofile/bad-bpf) - A collection of malicious eBPF programs that make use of eBPF's ability to read and write user data in between the usermode program and the kernel.
 - [TripleCross](https://github.com/h3xduck/TripleCross) - A Linux eBPF rootkit with a backdoor, C2, library injection, execution hijacking, persistence and stealth capabilities.
+- [Securing Kubernetes Workloads using LSM-BPF](https://ebpfchirp.substack.com/p/securing-kubernetes-workloads-using) - Learn how tools like KubeArmor leverage LSM-BPF to prevent malicious attacks such as Ransomware or Cryptojacking.
+- [Protecting HashiCorp Vault using KubeArmor](https://ebpfchirp.substack.com/p/protecting-k8s-secrets-using-kubearmor) - Learn how KubeArmor blocks malicious attacks on HashiCorp Vault and why use it instead of solutions like Tetragon or Falco.
```

Comment:
I have reservations about this link, and in particular about pushing KubeArmor over Tetragon for the wrong reasons:

> Tetragon, while capable of blocking threats, only does so after malicious actions have already occurred.

This is not correct. Tetragon blocks the syscall before it succeeds. I invite you to get in touch with the Tetragon folks for more details. See also https://tetragon.io/docs/concepts/enforcement/#override-return-value

(Disclaimer: I used to work at Isovalent)

My suggestion would be to remove this link from your PR for now, merge the rest of the PR, and we can discuss adding the link again after this point is fixed.